x-pack/plugins/maps/public/api/start_api.ts

```diff
@@ -9,7 +9,7 @@ import type { LayerDescriptor } from '../../common/descriptor_types';
 import type { CreateLayerDescriptorParams } from '../classes/sources/es_search_source';
 import type { SampleValuesConfig, EMSTermJoinConfig } from '../ems_autosuggest';
 import type { Props as PassiveMapProps } from '../lens/passive_map';
-import type { Props as MapProps } from '../react_embeddable/map_renderer';
+import type { Props as MapProps } from '../react_embeddable/map_renderer/map_renderer';
```

Comment:
super nit, exporting from an `index.ts` file in the `map_renderer` directory would make these imports appear less duplicative. but I'm fine either way.